libcontainer/configs/config.go

```diff
@@ -244,11 +244,13 @@ func (hooks Hooks) MarshalJSON() ([]byte, error) {
 
 // HookState is the payload provided to a hook on execution.
 type HookState struct {
-	Version    string `json:"ociVersion"`
-	ID         string `json:"id"`
-	Pid        int    `json:"pid"`
-	Root       string `json:"root"`
-	BundlePath string `json:"bundlePath"`
+	Version     string            `json:"version"`
+	ID          string            `json:"id"`
+	Status      string            `json:"status"`
+	Pid         int               `json:"pid"`
+	Root        string            `json:"root"`
+	BundlePath  string            `json:"bundlePath"`
+	Annotations map[string]string `json:"annotations"`
```

Comment:
Yep, I'm already doing this now, was sending this for a first glance in case someone is against this.

I was just thinking that the `Status` field is not very useful, because `prestart`,`poststart`,`poststop` happens at special time point and we don't need the status to identify container status, so I am also modifying the spec and trying to sync up HookState struct between spec and runc. :-)